CSharpFunctionalExtensions/Result/Error/ErrorList.cs

```diff
@@ -0,0 +1,24 @@
+﻿namespace CSharpFunctionalExtensions
+{
+    using System.Collections.Generic;
+
+    public sealed class ErrorList : List<Error>, ICombine
+    {
+        public ErrorList() { }
+        public ErrorList(IEnumerable<Error> errors) : base(errors) { }
+
+        public bool HasErrors => Count > 0;
+
+        public void Add(ErrorList ec) => AddRange(ec);
+
+
+        ICombine ICombine.Combine(ICombine value)
+        {
+            var errorList = new ErrorList((ErrorList)value);
```

Comment:
this seems dangerous to just cast to `ErrorList`

Reply:
Class deleted